action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
     required: true
     description: 'The project id'
   branch_name:
-    required: false
+    required: true
```

Comment:
Unrelated to the crux of the PR, it looks to me like this should be required?

Reply:
it shouldn't, Neon provides a default name that matches the id

Reply:
Ok I reverted this